config/deploy/01-metrics-server-tls-secret.yaml

```diff
@@ -0,0 +1,14 @@
+# This Secret is only for testing / dev.
+# This cert is valid till Jan 15 06:54:18 2032 GMT
```

Comment:
Can we use non-expiring cert? sounds like pain for future devs

Reply:
Sure :)